scrooge-generator/src/main/resources/scalagen/finagleServiceFunction.scala

```diff
@@ -15,6 +22,12 @@ addFunction("{{serviceFuncNameForWire}}", { (iprot: TProtocol, seqid: Int) =>
       }
 {{/exceptions}}
       case e => Future.exception(e)
+    } respond {
```

Comment:
the generated code looks like follow:
it would be better to keep '`} respond { `, or we need to change all styles accorrdingly
```
try {
      __stats_call.RequestsCounter.incr()
      val args = Call.Args.decode(iprot)
      iprot.readMessageEnd()
      (try {
        iface.call(args.name)
      } catch {
        case e: Exception => Future.exception(e)
      }) flatMap { value: Seq[Int] =>
        reply("call", seqid, Call.Result(success = Some(value)))
      } rescue {
        case e => Future.exception(e)
      } respond {
        case Return(_) =>
          __stats_call.SuccessCounter.incr()
        case Throw(ex) =>
          __stats_call.FailuresCounter.incr()
          __stats_call.FailuresScope.counter(Throwables.mkString(ex): _*).incr()
      ```